html/captive-portal/templates/login.html

```diff
@@ -70,8 +70,15 @@
           <div class="input">
             <span>[% i18n("Password") %]</span><input class="field" name="password" type="password" />
           </div>
+
           [% END %]
-          
+          [% FOREACH field IN mandatory_fields %]
+          <div class="input">
+            <span>[% i18n("$field") %]</span>
+            <input class="field" name="[%field%]" type="text" value="" /><br/>
```

Comment:
Yes but you know what i forgot to add the code behind that, so it just appear in the form but do nothing ;-)